examples/raytrace-parallel/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
Could this be removed? I'd prefer that nightly is explicitly used to build this example

Reply:
It's mostly a convenience - I found it a bit annoying to always remember to run `rustup run nightly ./build.sh` for build to work in this folder, but certainly can remove if you're strongly opposed.

Reply:
Reverted.